smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -813,7 +813,7 @@ def deviceHandler(evt) {
 		}
 	} else if (deviceInfo) {
 		if (deviceInfo.callbackUrl) {
-			sendToHarmony(evt, deviceInfo.callbackUrl)
+			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
```

Comment:
Do you have a good method to check for IP address in the URL? If not I can
dig for one.

On Mon, Jun 20, 2016, 4:42 PM Luke Bredeson <notifications@github.com>
wrote:

> In
> smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy
> <https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/1003#discussion_r67762346>
> :
>
> > @@ -813,7 +813,7 @@ def deviceHandler(evt) {
> >  		}
> >  	} else if (deviceInfo) {
> >  		if (deviceInfo.callbackUrl) {
> > -			sendToHarmony(evt, deviceInfo.callbackUrl)
> > +			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
>
> state.harmonyHubs is the new way of storing hub connection info, while
> state[evt.deviceId] is the old way. Both are checked, since we need to stay
> backwards compatible, but state.harmonyHubs is the "new" way. We should
> make sure we have the ability to call either method from either piece of
> data.
>
> If you use their cloud URL to make the determination instead of an IP
> address, you'll just need to be able to handle the case where they decide
> to change their cloud URL some day.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/1003/files/6f9d8b0a5f9b7d764e69523fdb2e7d28ef662d3b#r67762346>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA_0K_q2kbWeJV-yosPXUJvFe4dqc3mNks5qNvtMgaJpZM4I6C3d>
> .
>
-- 
Juan Pablo Risso
Washington, DC

<https://twitter.com/juano2310>
<https://plus.google.com/u/0/+JuanPabloRisso>
<https://www.youtube.com/user/juano2310>